tests/p2p/netfilter/test_factory.py

```diff
@@ -18,7 +18,8 @@ def test_factory(self):
         rule = NetfilterRule(match, NetfilterReject())
         pre_conn.add_rule(rule)
 
-        wrapped_factory = HathorServerFactory('testnet', PeerId(), node=None, use_ssl=False)
+        wrapped_factory = HathorServerFactory('testnet', PeerId(), node=None, use_ssl=False,
+                                              enable_sync_v1=True, enable_sync_v2=False)
```

Comment:
Should we test it with sync-v2 only and the bridge? If it doesn't matter, I feel we should add a comment explaining why it doesn't matter.

Reply:
Nope, I just forgot to fix this test case. I think I did this before making the base class with specialization for each case structure and forgot to update it. Anyway I fixed it to run in all 3 cases.